migration/v0.28.x-v0.29.x/upgrade-apps.md

```diff
@@ -1,13 +1,54 @@
 # Upgrade v0.28.x to v0.29.x
 
-## Steps
+## Prerequisites
+
+- [ ] Notify the users (if any) before the upgrade starts;
+- [ ] Check if there are any pending changes to the environment;
+- [ ] Check the state of the environment, pods, nodes and backup jobs:
+
+    ```bash
+    ./bin/ck8s test sc|wc
+    ./bin/ck8s ops kubectl sc|wc get pods -A -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,READY-false:status.containerStatuses[*].ready,REASON:status.containerStatuses[*].state.terminated.reason | grep false | grep -v Completed
+    ./bin/ck8s ops kubectl sc|wc get nodes
+    ./bin/ck8s ops kubectl sc|wc get jobs -A
+    velero get backup
+    ```
+
+- [ ] Silence the notifications for the alerts. e.g you can use [alertmanager silences](https://prometheus.io/docs/alerting/latest/alertmanager/#silences);
+
+## Steps that can be done before the upgrade - non-disruptive
+
+1. Pull the latest changes and switch to the correct branch:
+
+    ```bash
+    git pull
+    git switch -d v0.29.0
+    ```
 
 1. Update apps configuration:
 
     This will take a backup into `backups/` before modifying any files.
 
     ```bash
-    bin/ck8s init
+    ./bin/ck8s init
+    ```
+
+## Upgrade steps
+
+1. Upgrade starboard-operator
+
+    First upgrade the CRDs
+
+    ```bash
+    ./bin/ck8s ops kubectl wc apply -f ./helmfile/upstream/starboard-operator/crds/
+    ./bin/ck8s ops kubectl sc apply -f ./helmfile/upstream/starboard-operator/crds/
+    ```
+
+    Then upgrade the application
+
+    ```bash
+    ./bin/ck8s ops helmfile wc -l app=starboard-operator apply
+    ./bin/ck8s ops helmfile sc -l app=starboard-operator apply
     ```
```

Comment:
Does this have to be a separate step? It should be applied later with `./bin/ck8s apply {sc|wc}`

Reply:
No it doesn't have to be, I just thought it made sense to "complete" the upgrade of starboard in one go. I can remove this if you prefer not having it

Reply:
Yeah I think it is better to have fewer steps if possible.

Reply:
Let's have starboard be installed in the apply sc/wc step